src/__tests__/sdk_codes.test.ts

```diff
@@ -0,0 +1,53 @@
+import { Logger } from '../log';
```

Comment:
You're only using the type, so no need for a full import
```suggestion
import type { Logger } from '../log';
```